### ISSUE TITLE
Implementing fleet management components

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#26]
## User story
1. As a SD
2. I want to research the following modules for fleet management
- module that finds closest free robot
- mqtt/ vda5050 message function (the part of the module that speciies how the message looks likes and sends it
- route calculator module, that once the robot is chosen finds the path in the tree where the robot should go
- optional a different module for charging the robot, as that should probably be prioritiesed
3. So that I can later start implementing them

## Acceptance criteria
* Clarification on how to start implementing these modules

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by cat24max** [comment]
I recommend these modules:

- VMap Ingress module: Reads scanned LIDAR map file and converts it to FMS Python graph
- Graph module: Classes for Graph storage and functions for traversal (e.g., Dijkstra)
- VDA5050 module: Creates VDA5050 JSON strings, reads VDA5050 packets to dict (including validation)
- Webserver module: Receives and executes requests from UI
- MQTT Module: Establishes MQTT connection with broker, receives VDA5050 packets and forwards them to worker module. Sends outgoing VDA5050 packets.
- Worker module: Fulfills request async, plans and manages everything
